bower.json

```diff
@@ -12,7 +12,6 @@
     "iron-resizable-behavior": "polymerelements/iron-resizable-behavior#^1.0.0"
   },
   "devDependencies": {
-    "carbon-elements": "polymerelements/carbon-elements#gh-pages",
```

Comment:
I think we can just change this to
```
"carbon-elements": "git@github.com:PolymerElements/carbon-elements.git",
```
This seems to work for me given I have ssh key setup on my desktop.